python/lsst/sims/catUtils/supernovae/snObject.py

```diff
@@ -589,6 +590,94 @@ def SNObjectSED(self, time, wavelen=None, bandpass=None,
         SEDfromSNcosmo.addCCMDust(a_x=ax, b_x=bx, ebv=self.ebvofMW)
         return SEDfromSNcosmo
 
+    def SNObjectSourceSED(self, time, wavelen=None):
```

Comment:
I'm worried that, if someone does a dir() on SNObject, it is not going to be obvious from the name (SourceSed) that this returns the rest frame SED.